devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -153,6 +195,11 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 		def value = cmd.v1AlarmLevel == 255 ? "active" : cmd.v1AlarmLevel ?: "inactive"
 		result << createEvent(name: "alarm $cmd.v1AlarmType", value: value, isStateChange: true, displayed: false)
 	}
+
+	log.debug "isConfigured: ${isConfigured()}"
+	if (isEverspringSP817() && !isConfigured()) {
+		result += lateConfigure()
+	}
```

Comment:
sending a notificiationReport doesn't imply that the device is awake to receive commands, so this should only be in the wakeup handler

Reply:
Yes, but the wakeup events are infrequent and we needed to force quick configuration when the parameter is changed. And it worked, a configuration is really fast when after changing parameter user is provoking for example tamper event. So thanks to that we can be sure that the configuration will be complete after the very next motion event without user's awareness. And this change was merged previously, I just removed checking for Everpring SP815.

Reply:
Generally when a user updates a parameter they should press the wakeup button on the device. That typically produces the WakeUpNotification.

What does the device's user manual say about setting the configurations? Usually before the section listing the configuration options and valid values it will indicate that you need to perform some action on the device to tell the controller to send the updated values. (I believe for Fibaro devices this was usually three quick clicks of the button or pressing and holding until a certain color, and Aeotec devices were either one click of the button or three clicks of the button. So there should be something similar.)

I definitely wouldn't rely on squeezing in messages after receiving a notification report. I definitely see that you have it in the WUN handler, which is good. It might be best to remove the calls elsewhere and make sure that the device prefs page has a note indicating what the user should do after saving the device prefs changes. ("Consult the device manual for activating changes on the device." Or "Press the button on the device three times." if the manual states that. Or note both, since sometimes firmware changes on later devices...)

Reply:
There is no such information in the user manual but I checked if it works for pressing the tamper button and yes, there is a need to press it three times in order to complete the requested configuration. I will update the description in preferences to guide the user and remove messages after notification report for both devices and from the initialPoll() for SP815.